answerking_app/tests/test_integration/fixtures/orders/invalid-quantity-2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": 1,
+  "lineItems": [
+    {
+      "product": {
+        "id": 1
```

Comment:
maybe call this file "invalid-quantity-string" and the other one invalid-quantity-negative" just to differentiate them more easily